index.js

```diff
@@ -115,7 +114,11 @@ Funnel.prototype.processFilters = function(inputPath) {
   if (this.files && !this.exclude && !this.include) {
     files = this.files.slice(0); //clone to be compatible with walkSync
   } else {
-    files = walkSync(inputPath);
+    if (this.include && this.include.filter(function(a) { return a instanceof Minimatch; }).length > 0) {
```

Comment:
Since `this.includes` should not be changing after instantiation, and we already have a few methods to setup the include and exclude filters can't we stash a local `this._includeIncludesMinimatch` variable (instead of having to check on every rebuild)?